cdn-server/nginx.conf

```diff
@@ -29,9 +29,14 @@ rtmp {
             live on;
             dash on;
             dash_path /var/www/dash;
-            dash_fragment 3;
-            dash_playlist_length 60;
+            dash_fragment 4;
```

Comment:
What's the reason to change "dash_fragment" and "dash_playlist_length" parameters?

Reply:
dash_fragment 4; # 4 second is generaly a good choice for live
dash_playlist_length 120; # keep 120s of tail